core/primitives/src/views.rs

```diff
@@ -1206,6 +1212,10 @@ impl From<Action> for ActionView {
                 delegate_action: action.delegate_action,
                 signature: action.signature,
             },
+            Action::DeployGlobalContract(action) => {
+                let code = hash(&action.code).as_ref().to_vec();
+                ActionView::DeployGlobalContract { code, link_account: action.link_account }
```

Comment:
nit: Can you rename code in `DeployGlobalContract` to `code_hash` please? Or even better rename it and change the type to CryptoHash.